File: sources/api/Models/Payement/PaymentDTO.cs
using Microsoft.AspNetCore.Http;

namespace DotNetAPI.Models.Payment
{
    public class PaymentDTO
    {
        public int UserId { get; set; }
        public int UserInCreditId { get; set; }
        public int GroupId { get; set; }
        public float Amount { get; set; }
        public int? DebtAdjustmentId { get; set; }
        public DateTime PaymentDate { get; set; }
        public int? type { get; set; }
        public IFormFile? Image { get; set; }
    }
}

File: sources/mobile/app/src/main/java/com/console/ratcord/domain/entity/payment/PaymentDTO.kt
package com.console.ratcord.domain.entity.payment

import androidx.room.PrimaryKey
import kotlinx.serialization.Serializable
import kotlinx.serialization.Transient

@Serializable
data class PaymentDTO(
    val id: Int?,
    val userId: Int,
    val userInCreditId: Int,
    val groupId: Int,
    val amount: Float,
    val debtAdjustmentId: Int?,
    val paymentDate: String?, // issue take good date format
    val type: Int?,
    @Transient
    val imagePath: String? = null
)